src/server.js

```diff
@@ -94,7 +94,7 @@ const initApp = function () {
   app.post('/api/v2/status', fogController.getFogControllerStatusEndPoint);
   app.get('/api/v2/instance/create/type/:type', fogController.fogInstanceCreateEndPoint);
 
-  // app.get('/api/v2/instance/getfabriclist', fogController.getFogListEndPoint);
```

Comment:
Did you change clients that consume this endpoints????

Reply:
yes

Reply:
for example pureegde ?

Reply:
done